ckanext/dcat/tests/profiles/dcat_ap_2/test_scheming_support.py

```diff
@@ -839,6 +838,116 @@ def test_statement_literal(self):
         assert dataset["notes"] == "This is a dataset"
         assert dataset["access_rights"] == "Some statement"
 
+    def test_multiple_contacts(self):
+
+        data = """
+        @prefix dcat: <http://www.w3.org/ns/dcat#> .
+        @prefix dct: <http://purl.org/dc/terms/> .
+        @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+        @prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
+
+        <https://example.com/dataset1>
+          a dcat:Dataset ;
+          dct:title "Dataset 1" ;
+          dct:description "This is a dataset" ;
+            dcat:contactPoint [ a vcard:Kind ;
+                vcard:fn "Test Contact 1" ;
+                vcard:hasEmail <mailto:contact1@example.org> ],
+            [ a vcard:Kind ;
+                vcard:fn "Test Contact 2" ;
+                vcard:hasEmail <mailto:contact2@example.org> ] ;
+        .
+        """
+
+        p = RDFParser()
+
+        p.parse(data, _format="ttl")
+        datasets = [d for d in p.datasets()]
+
+        dataset = datasets[0]
+        assert len(dataset["contact"]) == 2
+        assert dataset["contact"][0]["name"] == "Test Contact 1"
+        assert dataset["contact"][0]["email"] == "contact1@example.org"
+        assert dataset["contact"][1]["name"] == "Test Contact 2"
+        assert dataset["contact"][1]["email"] == "contact2@example.org"
+
+    def test_multiple_publishers(self):
+
+        data = """
+        @prefix dcat: <http://www.w3.org/ns/dcat#> .
+        @prefix dct: <http://purl.org/dc/terms/> .
+        @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+        @prefix org: <http://www.w3.org/ns/org#> .
+        @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+        @prefix foaf: <http://xmlns.com/foaf/0.1/> .
+        @prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
+
+        <https://example.com/dataset1>
+          a dcat:Dataset ;
+          dct:title "Dataset 1" ;
+          dct:description "This is a dataset" ;
+          dct:publisher [ a org:Organization ;
+                    skos:prefLabel "Test Publisher 1" ;
+                    vcard:hasEmail <mailto:publisher1@example.org> ;
```

Comment:
Same here with for example: dct:identifier "https://orcid.org/0000-0002-9095-9201" ;